3_wtemp.yml

```diff
@@ -0,0 +1,29 @@
+target_default: 3_wtemp
+
+# include 1_wqpdata.yml mainly to get the lib.yml targets without reading those targets twice
```

Comment:
you are including `1_nwisdata.yml` here, so the comment is either outdated or the yml needs to be changed. I think you can get the `lib.yml` targets by depending on `1_nwisdata.yml` since it also depends on `1_wqpdata.yml`

Reply:
good point, thanks. fixed.